docs/pages/access-controls/guides/role-templates.mdx

```diff
@@ -110,7 +113,7 @@ traits `logins` and `kubernetes_groups`.
 Use `tctl` to create a role template:
 
 ```bash
-$ tctl create -f ~/scripts/access-controls/devs.yaml
+tctl create -f ~/scripts/access-controls/devs.yaml
```

Comment:
I think I accidentally leftover my local path, let's change to:

tctl create -f devs.yaml

Reply:
NP - removed!